packages/platforms/agema/x86-64/x86-64-agema-agc7648a/onlp/PKG.yml

```diff
@@ -1,5 +1,5 @@
 variables:
-  platform: x86-64-agc7648-r0
+  platform: x86-64-agc7648a-r0
```

Comment:
Yes. This is what i got. 

root@localhost:~#
root@localhost:~# onie-shell
EXT4-fs (sdb2): mounted filesystem with ordered data mode. Opts: (null)
/ # onie-sysinfo
x86_64-agc7648a-r0
/ #

it used to be x86_64-agc7648-r0 now  it is x86_64-agc7648a-r0